lib/features/game/views/widgets.dart

```diff
@@ -104,10 +114,23 @@ class ResultListItem extends StatelessWidget {
       child: SizedBox(
         height: 50.0,
         child: Row(
-          mainAxisAlignment: MainAxisAlignment.spaceAround,
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
```

Comment:
Do you need this? That does nothing when all of children are Expanded

Reply:
done